scripts/opx_build

```diff
@@ -17,7 +17,15 @@
 
 set -e
 
-DIST=jessie
+DIST=$(lsb_release --codename --short)
```

Comment:
Using the gbp wrapper may work fine for both Xenial and Jessie.  I'll try it later today.